java/src/jmri/jmrix/openlcb/OlcbStringIOManager.java

```diff
@@ -0,0 +1,98 @@
+package jmri.jmrix.openlcb;
+
+import java.util.*;
+
+import javax.annotation.Nonnull;
+
+import jmri.StringIO;
+import jmri.jmrix.can.CanSystemConnectionMemo;
+
+/**
+ * Implement a StringIOManager for OpenLCB StringIOs.
+ *
+ * @author Bob Jacobsen      Copyright (C) 2024
+ * @author Daniel Bergqvist  Copyright (C) 2020
+ */
+public class OlcbStringIOManager extends jmri.managers.AbstractStringIOManager {
+
+    // Whether we accumulate partially loaded objects in pendingStringIOs.
+    private boolean isLoading = false;
+    // Turnouts that are being loaded from XML.
+    private final ArrayList<OlcbStringIO> pendingStringIOs = new ArrayList<>();
+
+    public OlcbStringIOManager(CanSystemConnectionMemo memo) {
+        super(memo);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @Nonnull
+    public CanSystemConnectionMemo getMemo() {
+        return (CanSystemConnectionMemo) memo;
+    }
+
+    @Override
+    @Nonnull
+    public StringIO provideStringIO(@Nonnull String sName) throws IllegalArgumentException {
+        String name = sName.substring(getSystemPrefix().length()+1);
+        return new OlcbStringIO(getSystemPrefix(), name, (CanSystemConnectionMemo) memo);
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    @Nonnull
+    public StringIO provide(@Nonnull String name) throws IllegalArgumentException { 
+        return provideStringIO(name); 
+    }
+
+    @Override
+    @Nonnull
+    public StringIO createNewStringIO(String sName, String uName) {
```

Comment:
The user name is not being added to the new bean.

Reply:
Fixed.  Thanks.